index.js

```diff
@@ -50,7 +48,7 @@ const GATrack = {
       };
 
       gtag('event', action, payload);
-    } else if (typeof window.ga !== 'undefined') {
+    } else if (GATrack.isGA) {
```

Comment:
shouldn't we use the internal send method here too?

Reply:
Good catch.